backend/controllers/viewResidentialLocation.js

```diff
@@ -41,7 +43,7 @@ module.exports = ({ allocationService, elite2Api, keyworkerApi, complexityOfNeed
         const otherKeyworkers = keyworkerResponse.filter((keyworker) => keyworker.staffId !== offender.staffId)
         const formatNumberAllocated = (number) => (number ? `(${number})` : '')
         const isHighComplexity =
-          complexityOfNeed.enabled &&
+          isComplexityEnabledFor(activeCaseLoadId) &&
```

Comment:
as per psh, not sure the isComplexityEnabledFor call is needed?!

Reply:
You're right it's not required, will fix now :+1: